package/MDAnalysis/analysis/gnm.py

```diff
@@ -337,19 +341,25 @@ class closeContactGNMAnalysis(GNMAnalysis):
           the atoms that form a contact.
     MassWeight : bool (deprecated, optional)
           if set to ``True`` equivalent to `weights` set to "size".
-          .. Note:: This option does not perform a true mass weighting but
-                    weighting by the number of atoms in each residue; the name
-                    of the parameter exists for historical reasons and will
-                    be removed in 0.17.0. Until then, setting `MassWeight` to
-                    anything but ``None`` will override `weights`.
 
-    .. SeeAlso:: :class:`GNMAnalysis`
+    Notes
+    -----
+    The `MassWeight` option does not perform a true mass weighting but
+    weighting by the number of atoms in each residue; the name of the parameter
+    exists for historical reasons and will be removed in 0.17.0. Until then,
+    setting `MassWeight` to anything but ``None`` will override `weights`.
+
+    See Also
+    --------
+    :class:`GNMAnalysis`
+
 
     .. versionchanged:: 0.16.0
```

Comment:
I found by trial and error that the normal `.. *` sections only work in numpy docs when we leave two blank lines